meteor/server/api/blueprints/events.ts

```diff
@@ -189,7 +190,9 @@ export function reportPartInstanceHasStarted(cache: CacheForPlayout, partInstanc
 		})
 
 		cache.deferAfterSave(() => {
-			handlePartInstanceTimingEvent(cache.PlaylistId, partInstance._id)
+			Meteor.setTimeout(() => {
```

Comment:
I think this needs a comment explaining why there is a timeout here, as it is unclear for the casual reader (and me).